src/main/java/io/github/rabobank/shadow_tool/EncryptionService.java

```diff
@@ -1,21 +1,5 @@
 package io.github.rabobank.shadow_tool;
 
-import org.bouncycastle.util.encoders.Base64;
-
-import javax.crypto.Cipher;
-
-class EncryptionService {
-    private final Cipher cipher;
-
-    EncryptionService(final Cipher cipher) {
-        this.cipher = cipher;
-    }
-
-    String encrypt(final String value) {
-        try {
-            return Base64.toBase64String(cipher.doFinal(value.getBytes()));
-        } catch (final Exception e) {
-            throw new SecurityException(e);
-        }
-    }
+interface EncryptionService {
```

Comment:
Would it make sense to mark this as a functional interface? Also, let's add some documentation indicating that the return value of the encrypt method is directly logged, so make sure you properly encrypt what you don't want to expose (or possibly don't encrypt if you don't need/want it)

Reply:
I also would say the `DefaultEncryptionService` should have documentation, but can't blame you for this as it wasn't in the original version.